### PR TITLE
2020-2021 Technical Committee Election Results

### PR DESCRIPTION
**Pull request by alexander-hughes**
Signed-off-by: Alexander Hughes <Alexander.Hughes@pm.me>

### CODE REVIEW

File: src/pages/blog/2020-airship-tc-election.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+date: 2020-06-22T12:00:00.000Z
+category: 
+  - id: category-A7xLnYrE1
+    label: Election
+---
+
+The Airship team is excited to announce the completion of its second [_Technical Committee election_](
+http://lists.airshipit.org/pipermail/airship-discuss/2020-June/001005.html). The [_Technical Committee_](
+https://opendev.org/airship/governance#user-content-technical-committee), one of the two governing bodies for Airship,
+is responsible for ensuring Airship projects adhere to the project's core principles, promote standardization, and
```
Comment (abailey011) [inline src/pages/blog/2020-airship-tc-election.md:14]: Would be nice to be able to link to the project's core pinciples here.

File: src/pages/blog/2020-airship-tc-election.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+<br>
+
+We want to extend our congratulations to the new Technical Committee members and recognize the contributions of Andrew
+Leasck, James Gu, and Ryan van Wyk to the Airship Community through their roles on the Technical Committee last year.
+Governance by community-elected officials reflects Airship's continued dedication to the [_Four Opens_](
+https://osf.dev/four-opens/). The Technical Committee is organizing its schedule for the upcoming year. Once finalized,
+the Technical Committee will share the meeting schedule for 2020-2021 on the [_Wiki_](
+https://wiki.openstack.org/wiki/Airship/Airship-TC), where previous meeting agendas and recordings are also available.
+
+The Technical Committee is just one of the two governing bodies within the Airship community. With that election wrapped
```
Comment (abailey011) [inline src/pages/blog/2020-airship-tc-election.md:34]: The first sentence here is a bit redundant, you mention this above.  I'd start this paragraph with "With the Technical Committee election wrapped up, they have turned..."

File: src/pages/blog/2020-airship-tc-election.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+up, they have now turned their attention to the Working Committee election. The [_Working Committee_](
+https://opendev.org/airship/governance#user-content-working-committee) guides the project strategy, helps arbitrate when
+there is a disagreement between Core Reviewers within a single project or between Airship projects, defines core project
+principles, assists in marketing and communications provides product management, and offers ecosystem support.
+
+Nominations and elections for the Working Committee will take place next month. Anyone who has contributed to the
+Airship project within the last 12 months is eligible to run for the Working Committee and vote in the upcoming
```
Comment (abailey011) [inline src/pages/blog/2020-airship-tc-election.md:41]: Might be worth noting somehow that the WC requirements are slightly different than the TC requirements.

File: src/pages/blog/2020-airship-tc-election.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+Governance by community-elected officials reflects Airship's continued dedication to the [_Four Opens_](
+https://osf.dev/four-opens/). The Technical Committee is organizing its schedule for the upcoming year. Once finalized,
+the Technical Committee will share the meeting schedule for 2020-2021 on the [_Wiki_](
+https://wiki.openstack.org/wiki/Airship/Airship-TC), where previous meeting agendas and recordings are also available.
+
+The Technical Committee is just one of the two governing bodies within the Airship community. With that election wrapped
+up, they have now turned their attention to the Working Committee election. The [_Working Committee_](
+https://opendev.org/airship/governance#user-content-working-committee) guides the project strategy, helps arbitrate when
+there is a disagreement between Core Reviewers within a single project or between Airship projects, defines core project
+principles, assists in marketing and communications provides product management, and offers ecosystem support.
```
Comment (abailey011) [inline src/pages/blog/2020-airship-tc-election.md:38]: Think you missed a comma here.

File: src/pages/blog/2020-airship-tc-election.md (hunk 1 above)
Comment (alexander-hughes) [reply src/pages/blog/2020-airship-tc-election.md:14]: Agreed, let's take this to the next WC meeting who is responsible for defining the project core principles and get this added to governance.

File: src/pages/blog/2020-airship-tc-election.md (hunk 2 above)
Comment (alexander-hughes) [reply src/pages/blog/2020-airship-tc-election.md]: Updated.

File: src/pages/blog/2020-airship-tc-election.md (hunk 4 above)
Comment (alexander-hughes) [reply src/pages/blog/2020-airship-tc-election.md]: Great catch!  Reworked slightly.

File: src/pages/blog/2020-airship-tc-election.md (hunk 3 above)
Comment (alexander-hughes) [reply src/pages/blog/2020-airship-tc-election.md]: Clarified WC requirements and addressed error with who is eligible to vote.

**Pull request merged**